src/components/CampaignDetails.vue

```diff
@@ -49,19 +50,28 @@ export default {
       return this.$store.getters.getSingleCampaign(this.address)
     }
   },
-  created() {
-    const campaignAddress = "0xe8a0980C2B37C2C4FCE45e579301B00CC824bCFc"
-    this.$store.dispatch('fetchCampaign', campaignAddress)
-  },
   methods: {
+    /* eslint-disable */
     async submitContribution() {
-      // const campaignAddress = "0xe8a0980C2B37C2C4FCE45e579301B00CC824bCFc"
-      if (this.contribution <= this.campaign.min_contribution) {
-        this.userMessage = true;
-        return
-      }
       const campaignInstance = await ethCampaign.at(this.address);
-      campaignInstance.contribute({ from: this.$store.state.accountNum, value: this.contribution })
+      const result = await campaignInstance.contribute({
+        from: this.$store.state.accountNum,
+        value: web3.utils.toWei(this.contribution, "ether")
+      })
+      if (result && (this.contribution <= this.campaign.min_contribution)) {
+        try {
+          await axios.put(
+            `${VUE_APP_API_URL}campaigns/${this.campaign.id}`,
+            { contributors: (this.campaign.contributors + 1) }
+          );
+        } catch (error) {
+          return { error };
+        }
+      }
+    },
```

Comment:
Feel free to refactor or request changes on this, still not totally comfortable making changes on the FE

Reply:
This looks pretty good to me, I'm a little unsure of how JS would make that string comparison on line 61.